Student.java

```diff
@@ -70,4 +70,15 @@ public String changeSchool(String newSchool) {
 		return majorsMinors;
 	}
 	
+	public void displayAllInfo() {
+		System.out.println("Name: " + this.getName());
+		System.out.println("School: " + this.getSchool());
+		System.out.println("Majors/Minors: " + this.getMajorsMinors());
+		System.out.println("Graduation Year: " + this.getGradYear());
+		System.out.println("Initiation Year: " + this.getInitYear());
+		System.out.println("Brother is active: " + this.getIsActive());
+		System.out.println("Brother joined in the fall: " + this.getIsFall());
```

Comment:
Make if statements here and say "eligibility for bid vote: eligible" and "Joined: Fall (or Spring if false)"